vendor-onboarding-checklist.md

```diff
@@ -0,0 +1,148 @@
+<li><p>Create, read, update docs on SharePoint</p></li>
+<li><p>Read and update Trello</p></li>
+<li><p>Read and post messages to Slack</p></li>
+<li><p>Read GitHub repositories and comment on issues</p></li>
+<li><p>Participate in Mural sprint retrospectives</p></li>
+</ul></td>
+</tr>
+<tr class="even">
+<td>Project manager</td>
+<td>{none}</td>
+<td><ul>
+<li><p>Read and update work-items and PR comments in VSTS</p></li>
+</ul></td>
+</tr>
+<tr class="odd">
+<td>Technical lead</td>
+<td><ul>
+<li><p>Alaska Development Environment (MCI Person Service Dev endpoint, EIS Program Data Service Dev endpoint, ARIES Program Data Service Dev endpoint, Other TBD)</p></li>
+</ul></td>
+<td><ul>
+<li><p>Create, read, update work-items</p></li>
+<li><p>Push branches to VSTS hosted Git repos</p></li>
+<li><p>Create and comment on Git pull-requests</p></li>
+<li><p>Read build output</p></li>
+<li><p>Access various non-production services (MCI, EIS, ARIES) to perform development work</p></li>
+</ul></td>
+</tr>
+<tr class="even">
+<td>Developers</td>
+<td><ul>
+<li><p>Alaska Development Environment (MCI Person Service Test, EIS Program Data Service Test, ARIES Program Data Service Test, Other TBD)</p></li>
+</ul></td>
+<td><ul>
+<li><p>Create, read, update work-items</p></li>
+<li><p>Push branches to VSTS hosted Git repos</p></li>
+<li><p>Create and comment on Git pull-requests</p></li>
+<li><p>Read build output</p></li>
+<li><p>Access various non-production services (MCI, EIS, ARIES) to perform development work</p></li>
+</ul></td>
+</tr>
+<tr class="odd">
+<td>Researcher</td>
+<td>?</td>
+<td></td>
+</tr>
+<tr class="even">
+<td>User experience designer</td>
+<td>?</td>
+<td></td>
+</tr>
+<tr class="odd">
+<td>Visual designer</td>
+<td>?</td>
+<td></td>
+</tr>
+</tbody>
+</table>
+
+# Access procedures
+
+This section outlines procedures to grant access. Access to DHSS managed
+resources must be formally requested, sponsored and granted. All vendor
+access requests must be submitted through the EIS-R PMO. The information
+supplied below serves as a reference for how specific types of access
+request are granted once approved.
+
+| **Resource**                   | **Requested and granted via** |
+| ------------------------------ | ----------------------------- |
+| State of Alaska LDAP account   | DHSS IT Help Desk             |
+| State of Alaska VPN            | DHSS IT Help Desk             |
+| DHSS domain account            | DHSS IT Help Desk             |
+| DHSS SharePoint                | DHSS EIS-R PMO                |
+| Visual Studio Team Services    | DHSS Business Applications    |
+| Trello                         | TTS/18F                       |
+| Slack                          | TTS/18F                       |
+| Alaska DHSS GitHub             | DHSS Business Applications    |
+| Mural                          | TTS/18F                       |
+| Alaska Development Environment | DHSS Business Applications    |
+

```

Comment:
We may want to consider adding a section with required forms and supporting documentation to be submitted.

Non-State employees are required to submit the Security Agreement (located on internal DPAWeb), along with supporting documentation (verifying identity) and HIPAA training certificate.

Reply:
I agree - getting as much of the required forms and paperwork into one place will help the winning vendor hit  the ground running.

Reply:
Yes, I think it's a good idea to have the forms.  If we can make time for it, I'd like to have a pre-built package of forms, or checklist pointing to the forms for each role.  That would make it so vendor folks know exactly what they need to complete, even if they do need some help.  I think some forms have default routing instructions on them.  So, we'll just need to be extra clear that the forms need to route through EISR PMO.

Reply:
For now we could just start with a simple list of the required documents